sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -1180,13 +1180,10 @@ CosmosAsyncClient buildAsyncClient(boolean logStartupInfo) {
         stopwatch.start();
 
         if (Configs.shouldOptInDefaultCircuitBreakerConfig()) {
-            System.setProperty(
-                "COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG",
-                "{\"isPartitionLevelCircuitBreakerEnabled\": true, "
-                    + "\"circuitBreakerType\": \"CONSECUTIVE_EXCEPTION_COUNT_BASED\","
-                    + "\"consecutiveExceptionCountToleratedForReads\": 10,"
-                    + "\"consecutiveExceptionCountToleratedForWrites\": 5,"
-                    + "}");
+
+            if (Configs.shouldOptInDefaultCircuitBreakerConfig()) {
```

Comment:
then I am wondering do we need to introduce a new system environment variable? or this new variable is only targeted for test?

Reply:
Only targeted for test - setting a JSON property with all the weird escape patterns never seemed to work or I ran out of patience lol.

Reply:
Synced up offline - concluded to keep the change as is.